website/search/elastic_search.py

```diff
@@ -338,7 +338,7 @@ def update_node(node, index=None, bulk=False):
             'parent_id': parent_id,
             'date_created': node.date_created,
             'license': serialize_node_license_record(node.license),
-            'primary_institution': node.primary_institution.name if node.primary_institution else None,
+            'affiliated_institutions': [inst.name for inst in node.affiliated_institutions] if node.affiliated_institutions else [],
```

Comment:
Minor, but if `node.affiliated_institutions` returns `[]`, `[inst.name for inst in node.affiliated_institutions]` will already be `[]`:

```
In [1]: [i.some_attr for i in []]
Out[1]: []
```